### PR TITLE
chore: moving e2e jobs to monitor-ci

### PR DESCRIPTION
**Pull request by genehynson**
Closes https://github.com/influxdata/ui/issues/1112

This PR moves all our CI jobs that depend on private repos (monitor-ci and k8s-idpe) to a new CircleCI pipeline defined in the monitor-ci repo (see monitor-ci PR below). Included in this PR is a script that uses Circle's APIs to start and watch the monitor-ci "private" pipeline. See the "script summary" in the bash script included in this PR for more information on how it works. Some examples are included below.

Merge monitor-ci PR first: https://github.com/influxdata/monitor-ci/pull/207

Example of a failing job and copying artifacts: https://app.circleci.com/pipelines/github/influxdata/ui/3624/workflows/842b17a1-83a2-466d-928c-a050245c9415/jobs/28086/steps

Example of success: https://app.circleci.com/pipelines/github/influxdata/ui/3567/workflows/125456d5-53de-4e79-aafb-983366d14781/jobs/27570

Example of _quick_ success after retrying monitor-ci pipeline: https://app.circleci.com/pipelines/github/influxdata/ui/3657/workflows/76f40a6e-5e50-4e4e-93ae-dc7d65ba76c9/jobs/28382

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 46.5 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?vars%5Blighthouse_score_branch%5D=1073)

File: scripts/run-monitor-ci-tests.bash [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+						# print each artifact text and link
+						for url in "${artifacts_urls[@]}"; do
+							path=$(echo ${artifacts} | jq --arg url "${url}" 'map(select(.url == $url)) | .[].pretty_path')
+							printf '\n- %s\n' "${path}"
+							printf '   - URL: %s\n' "${url}"
+
+							# download artifact
+							filename=$(basename "${path}")
+							filename="${filename::-1}" # removes extra " from end
+							curl -L --output "monitor-ci/test-artifacts/results/${name}/${filename}" --request GET \
```
Comment (drdelambre) [inline scripts/run-monitor-ci-tests.bash:157]: lol

File: scripts/run-monitor-ci-tests.bash [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+				printf "https://app.circleci.com/pipelines/github/influxdata/monitor-ci/${pipeline_number}/workflows/${workflow_id}"
+			fi
+		done
+
+		exit $is_failure
+	fi
+
+	# sleep 1 minute and poll the status again
+	attempts=$(($attempts+1))
+	remaining_attempts=$(($max_attempts-$attempts))
```
Comment (drdelambre) [inline scripts/run-monitor-ci-tests.bash:174]: what, no exponential decay on your backoff strategy? nah... thanks for adding this

**Review by drdelambre — APPROVED** [review]
seems additive, so not stressing

**Pull request merged**